soh/src/code/z_play.c

```diff
@@ -472,14 +478,24 @@ void Play_Init(GameState* thisx) {
     gPlayState = play;
     //play->state.gfxCtx = NULL;
     uintptr_t zAlloc;
+    uintptr_t stairsAlloc;
     uintptr_t zAllocAligned;
+    uintptr_t stairsAllocAligned;
     size_t zAllocSize;
+    size_t stairsAllocSize;
     Player* player;
     s32 playerStartCamId;
     s32 i;
     u8 tempSetupIndex;
     s32 pad[2];
 
+    // #Region [SoH] Stairs
+    if (CVarGetInteger("gSimulateHeap", 0)) {
+        CVarSetInteger("gStairs", 1);
```

Comment:
By placing this in Play_Init, changes to the checkbox in the UI won't apply until the next time you change areas, since that's when Play_Init is run, right?  It might be nice to mention that in the tooltip that that checking or unchecking the box won't apply until the next area load.

Reply:
Yes will add that to the tooltip!